blog/Introducing-Feliz-v1.0.md

```diff
@@ -0,0 +1,39 @@
+- title: Introducing Feliz v1.0
+- subtitle: Announcing the official stable release of Feliz
+
+## Introducing Feliz v1.0
+
+Today, we are happy to announce the official stable release of [Feliz](https://github.com/Zaid-Ajaj/Feliz). This comes after 10 months of development since the initial version and after more than 100 packages in between where we kept adding support for more and more React APIs as well as increasing the API coverage for Html, SVG and CSS elements.
```

Comment:
For me, 100 packages is a bit confusing because I would expect 100 independent packages/libraries.

Perhaps, saying "100 releases" would be less misleading ?